games-emulation/libretro-meta/libretro-meta-9999-r5.ebuild

```diff
@@ -39,6 +39,7 @@ RDEPEND="
 	mednafen-psx? ( games-emulation/mednafen-psx-libretro )
 	mednafen-psx-hw? ( games-emulation/mednafen-psx-hw-libretro )
 	mupen64? ( games-emulation/mupen64next-libretro )
+	pcsx-rearmed? ( games-emulation/pcsx-rearmed-libretro )
```

Comment:
Please re-add the flag in `metadata.xml`.

Reply:
I was wondering whether it is required to bump the revision of the libretro-meta ebuild when adding previously unsupported use flags. Nothing changes for the existing use flags. People who have the meta ebuild already installed would pick up the change because they would enable the new use flag.

Not bumping it simplifies maintenance for yet-to-be-merged branches.